src/core/toga/widget_registry.py

```diff
@@ -0,0 +1,32 @@
+class WidgetRegistry:
```

Comment:
Didn't pick this up on my first review, but I don't think this needs to be a top-level module holding a single class. It fits just as well in `toga.widgets.base` as a core feature of the base widget implementation.

Reply:
(to be clear - I mean as a class in `base.py`, not some sort of inner class on the base widget)